packages/@aws-cdk/aws-codedeploy/test/ecs/integ.deployment-config.ts

```diff
@@ -0,0 +1,19 @@
+import * as cdk from '@aws-cdk/core';
+import * as integ from '@aws-cdk/integ-tests';
+import * as codedeploy from '../../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-codedeploy-ecs-config');
+
+new codedeploy.EcsDeploymentConfig(stack, 'LinearConfig', {
```

Comment:
Can you update this integ test to perform an actual ecs deployment? I know it will probably require
manual steps, but that will help us ensure that we are testing an actual scenario.

Reply:
Yes, though that might be more useful in my follow-up PR with an L2 construct for deployment groups.  These changes are dependent on this PR, so I havent put it up yet:
https://github.com/clareliguori/aws-cdk/commit/7d29f934e72057c4606e4f8438565461ec82ccd0